# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======

RobbieBaseActionGoal action_goal
RobbieBaseActionResult action_result
RobbieBaseActionFeedback action_feedback
